.travis.yml

```diff
@@ -13,4 +13,5 @@ install:
   - make init
 script:
   - make build test lint coverage
+  - make docker-all
```

Comment:
Should this be _replacing_ what's currently done in Travis instead of just being added to it? (Btw, if yes, Travis `langue:` can change from `go` to `generic` as go-specific features will no longer be required inside the Travis VM.